romana-install/roles/romana/install-agent/templates/config/agent_proxy_add_policy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+body=`echo $1 | cut -d'=' -f2`
+echo Posting $body to http://{{ romana_master_ip }}:9630/
```

Comment:
It communicates with `policy-agent` which runs on all the hosts and listens on all the interfaces - so localhost is fine.

Policy agent meant to become a true service one day so it will be discoverable and so on.